app/controllers/api/v1/accounts_controller.rb

```diff
@@ -88,7 +88,7 @@ def check_account_confirmation
   end
 
   def relationships(**options)
-    AccountRelationshipsPresenter.new([@account.id], current_user.account_id, **options)
+    AccountRelationshipsPresenter.new([@account], current_user.account_id, **options)
```

Comment:
For the single-account calls -- the presenter calls `to_a` on what it receives, so we could skip the array wrap in the calls.

Reply:
It wouldn't, `Account#to_a` does not exist. The `to_a` is to make sure we have an array and not a collection that would have us perform multiple SQL queries.